cypress/PageObjects/SendMoneyPage.js

```diff
@@ -92,9 +92,9 @@ class SendMoneyPage {
     return cy.get('[role="button"]').contains('Cancel link')
   }
 
-  get sendAddressButton() {
-    return cy.contains('Send to address')
-  }
+  // get sendAddressButton() {
```

Comment:
@MathiasMoralesDixtra you just removed the method but did not disable the test that actually use it
  it('User is able to send money from a new wallet using address', () => {
replace it
by 'xit' and uncomment the sendAddressButton method